src/plugins/expressions/common/execution/types.ts

```diff
@@ -84,4 +84,5 @@ export interface ExecutionContextSearch {
   filters?: Filter[];
   query?: Query | Query[];
   timeRange?: TimeRange;
+  sessionId?: string;
```

Comment:
I think it's probably best if we stick to `searchSessionId` here and below. I think it's more explicit.